tests/codegen/common.h

```diff
@@ -32,7 +53,20 @@ static void test(
   ASSERT_EQ(driver.parse_str(input), 0);
 
   ClangParser clang;
-  clang.parse(driver.root_, bpftrace);
+  // TODO (mmarchini): don't use kernel flags on tests
```

Comment:
Couldn't get the `args_multiple_tracepoints` tests to work without kernel headers. We can probably rewrite those tests to use tracepoints that don't rely on kernel types, but this can be done in the future, it's better to have those tests running first.

Reply:
Ok, tests won't work on CI with this. I'll have to try some alternatives.

Reply:
Is this stuff just to get the `umode_t` type in? Could just remove the `mode` entry from the tracepoint format definitions you've embedded in the test to save on a lot of extra complexity here - there's no need for them to match the real kernel tracepoint definitions

Reply:
Even if we change umode_t, size_t and all struct pointers to basic types, we still get:

```
definitions.h:8:3: error: unknown type name 's64'
definitions.h:10:3: error: unknown type name 'u64'
definitions.h:11:3: error: unknown type name 'u64'
definitions.h:22:3: error: unknown type name 's64'
definitions.h:24:3: error: unknown type name 'u64'
definitions.h:25:3: error: unknown type name 'u64'
definitions.h:35:3: error: unknown type name 's64'
definitions.h:37:3: error: unknown type name 'u64'
```

Because of https://github.com/iovisor/bpftrace/blob/master/src/tracepoint_format_parser.cpp#L166-L181. Suggestions?